be/src/Project/Services/Ordering/Ordering.Api/Services/OrdersService.cs

```diff
@@ -166,9 +166,11 @@ public async Task<PagedResults<IEnumerable<OrderServiceModel>>> GetAsync(GetOrde
                 orders = orders.Where(x => x.SellerId == model.OrganisationId.Value);
             }
 
-            if (!string.IsNullOrWhiteSpace(model.SearchTerm))
+            if (string.IsNullOrWhiteSpace(model.SearchTerm) is false)
             {
-                orders = orders.Where(x => x.ClientName == model.SearchTerm || x.OrderItems.Any(y => y.ExternalReference == model.SearchTerm));
+                orders = orders.Where(x => x.ClientName.ToLower().StartsWith(model.SearchTerm.ToLower()) 
```

Comment:
@SebaKas21 , now it's truly case insensitive. When order item has an external reference "ORD_01" and user searches for "ord", the right results are retrieved.